noxfile.py

```diff
@@ -6,6 +6,19 @@
 import shutil
 import nox
 
+@nox.session(py="3")
+def translation(session):
+    session.install("-r", "requirements.txt")
+    target_dir = "locales"
+    session.run(
+        "sphinx-build", 
+        "-b",  # select a builder
+        "gettext",  # build gettext-style message catalogs (.pot file)
```

Comment:
Let's keep the option names coupled with their values
```suggestion
        "-b", "gettext",  # build gettext-style message catalogs (.pot file)
```